src/client/registry.ts

```diff
@@ -1,9 +1,12 @@
 import { MapDocumentNode } from '@superfaceai/ast';
+import createDebug from 'debug';
 import * as zod from 'zod';
 
 import { isProviderJson, ProviderJson } from '../internal';
 import { HttpClient } from '../lib/http';
 
+const registryDebug = createDebug('superface:Registry');
```

Comment:
i think all other are lowercase, let's keep it the same everywhere

Reply:
At least on is upper cased https://github.com/superfaceai/one-sdk-js/blob/main/src/client/profile-provider.ts#L49
I will fix it to lower case everywhere :)